service/src/main/resources/db/migration/V329__service_need_shift_care_enum.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TYPE shift_care_type AS ENUM (
+    'FULL',
+    'INTERMITTENT',
+    'NONE'
+);
+
+ALTER TABLE service_need
+    ADD COLUMN shift_care_temp shift_care_type NOT NULL DEFAULT 'NONE'::shift_care_type;
+
+ALTER TABLE service_need DISABLE TRIGGER set_timestamp;
```

Comment:
Trigger is disabled to prevent launching unnecessary Varda-updates for all children in shift care. Is there any risk for launching other automated operations based on a detected service need change?